src/mapeo-project.js

```diff
@@ -34,8 +43,12 @@ export class MapeoProject {
    * @param {Buffer} opts.projectKey 32-byte public key of the project creator core
    * @param {Buffer} [opts.projectSecretKey] 32-byte secret key of the project creator core
    * @param {Partial<Record<import('./core-manager/index.js').Namespace, Buffer>>} [opts.encryptionKeys] Encryption keys for each namespace
+   * @param {ProjectSettingsConfig} opts.projectSettingsConfig
```

Comment:
This name doesn't feel right. We might well use this indexWriter and the db for other things in the future. I suggest:

```js
/**
 * @param {Database} opts.sharedDb
 * @param {IndexWriter} opts.sharedIndexWriter
 */
```

Reply:
addressed via https://github.com/digidem/mapeo-core-next/pull/187/commits/6c5822ad6c6654925f8629649c7c7d97c7661dab